pages/2_DataRAG.py

```
@@ -11,7 +13,59 @@
 datarag_paths = [datarag_assets_path, datarag_results_path]
 
 datarag_paths_selection = st.selectbox("Select DataRAG path", datarag_paths)
+file_format='delta'
 
 if st.button("Preview Data"):
```

Ruff (0.8.2)

`st` may be undefined, or defined from star imports

(F405)